.github/workflows/update.yml

```diff
@@ -43,14 +44,20 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-22.11
 
-      # - uses: cachix/cachix-action@v12
-      #   with:
-      #     name: acvm-simulator-cache
-      #     authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+      - name: Build with Nix
+        run: |
+          nix build github:noir-lang/acvm-simulator/jb-nix-flake
 
-      - name: Build environment and Compile
+      - name: Resolve symbolic link
         run: |
-          nix-shell acvm-simulator.nix --pure --run "./build.sh"
+          realpath ./result > result_path.txt
+          echo "RESULT_PATH=$(cat result_path.txt)" >> $GITHUB_ENV
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: release-artifact
+          path: ${{ env.RESULT_PATH }}
```

Comment:
would symlink not work here? if not would `RESULT_PATH=$(readlink result)` be easier?

Reply:
Fixed